Samples/src/FSharpWithQSharp/FsharpDriver/FsharpDriver.fsproj

```
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="Driver.fs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.5.1902.2802" />
```

```suggestion
    <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.5.1903.2902" />
```

This project may not even need a direct dependency on the Quantum Development Kit package, as that should be included by the project reference at L17 below?

---

Indeed this reference is not required; I'll remove it.